CHANGELOG.rst

```diff
@@ -2,9 +2,14 @@
 Release Notes
 =============
 
-Unreleased
+1.18.0
 ================
 
+- AP-2106 Add new GPG keys for APT and RPM repositories signature rotation (#44)
```

Comment:
I'd remove the `AP-2106`, it's useful for internal tracking of PRs, but doesn't need to end up in the changelog.